src/engine/Settings.cs

```diff
@@ -2,11 +2,14 @@
 using Newtonsoft.Json;
 
 /// <summary>
-///   Main object for containing player changeable game settings
+///   Class that handles storing and applying player changeable game settings.
 /// </summary>
 public class Settings
 {
-    private static readonly Settings SingletonInstance = LoadSettings();
+    /// <summary>
+    ///   Singleton used for holding the live copy of game settings.
+    /// </summary>
+    private static readonly Settings SingletonInstance = LoadInstance();
```

Comment:
I think if you named the method `LoadSettings` git wouldn't show this as a change.

Reply:
I can change it, but is there a reason that matters really?

Reply:
The less changes there are make it easier to review a PR as there's less diff to look at.